.github/workflows/ci.yaml

```diff
@@ -14,6 +14,8 @@ jobs:
       requiredJobs: ${{ steps.interrogate.outputs.requiredJobs }}
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Why not 2 or some small number? Seems like this does the entire repo?

Reply:
@tbpg I tried a variety of other values and was consistently getting an error that HEAD~ was ambiguous. The docs indicate that setting `0` "Fetch all history for all tags and branches".